docs/framework/guides/tutorials/implementing-a-block-widget.md

```diff
@@ -902,7 +902,7 @@ export default class SimpleBoxEditing extends Plugin {
 	}
 
 	_defineConverters() {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.